docs/spec-files/requirements.md

```diff
@@ -2,7 +2,7 @@
 
 The [DIDComm Messaging](.) design attempts to be:
 
-1. **Secure** (tamper-proof; uses best-of-breed crypto; allows parties to talk both on and off the record.)
+1. **Secure** (specifically, MUST have message confidentiality,message integrity, and message authentication; MUST use best-of-breed crypto; MUST allows parties to emit both repudiable and non-repudiable messages.)
```

Comment:
I think we need explain `Confidentiality` more here. What exactly it means.

Just an example. We define non-repudiation layer without clear message is it explicitly intended for non-repudiation or in some cases for auth too. Unintended non-repudiation looks like confidentiality issue.

Reply:
Okay, I have updated the language in an attempt to make it clearer. Do you like it better now?